clusterloader2/testing/load/dnsperf-deployment.yaml

```diff
@@ -0,0 +1,71 @@
+{{$HostNetworkMode := DefaultParam .CL2_USE_HOST_NETWORK_PODS false}}
+{{$CpuRequest := DefaultParam .CpuRequest "10m"}}
```

Comment:
This looks like copy paste of deployment.yaml file from https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/load/deployment.yaml

I we expect this file to be drop-in replacement for deployment.yaml it must be compatible with it (so have the same labels, resource requests, taints etc). I'm afraid that keeping two files with very similar content isn't the best approach and sooner or later we will forget updating one of them.

How about merging those files together? I think the content looks quite similar, only container and configmaps sections are different.

Reply:
Good point, let me try to merge them.